cernopendata/templates/cernopendata_theme/header.html

```diff
@@ -18,6 +18,15 @@
         </div>
       </form>
   </div>
+  <div class="checkbox-ondemand">
+    <form>
+        {% set checked_value = request.args.get('f','').split(':')[-1] %}
+        <input onChange="this.form.submit()" type="checkbox" name="availability" value="{{checked_value}}"
+            {% if checked_value == 'ondemand' %}checked{% endif %}
+            aria-label="Include on-demand datasets">
```

Comment:
- "include" with lowercase "I"
- please also put a space after search box so that the `[ ] include...` would be a bit more separated